src/components/form-elements/draft-js-mention-selector/createMentionSelectorState.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+import { ContentState, EditorState, Modifier, SelectionState } from 'draft-js';
+import DraftMentionDecorator from './DraftMentionDecorator';
+
+// returns data for first mention in a string
+const getMentionFromText = (text: string) => {
+    // RegEx.exec() is stateful, so we create a new regex instance each time
+    const mentionRegex = /([@＠﹫])\[([0-9]+):([^\]]+)]/gi;
```

Comment:
nit: could we pull this out of the function so that we don't repeatedly compile the regex. Luckily, i don't think this will run all that often, so not necessary if possible

Reply:
No, see comment